eventing/samples/cronjob-source/README.md

```diff
@@ -0,0 +1,58 @@
+# Cron Job Source example
+
+Cron Job Source example shows how to configure Cron Job as event source for functions.
+
+## Deployment Steps
+
+### Prerequisites
+
+1. Setup [Knative Serving](https://github.com/knative/docs/tree/master/serving).
+1. Setup
+   [Knative Eventing](https://github.com/knative/docs/tree/master/eventing).
+
+### Channel
```

Comment:
Do we need to use a Channel here? Feedback on the GitHub source suggested that pointing directly at a Knative service was easier for simple cases: https://github.com/knative/docs/tree/master/eventing/samples/github-source#create-event-source-for-github-events

Reply:
Updated to point to service directly.